python/paddle/v2/framework/tests/test_l1_norm_op.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import unittest
+from op_test import OpTest
+
+
+class TestL1NormOp(OpTest):
+    """Test l1_norm
+    """
+
+    def setUp(self):
+        self.op_type = "l1_norm"
+        self.max_relative_error = 0.05
```

Comment:
why the max_relative_error is so high

Reply:
It might be a typo. Fixed in 19a8c01b427a17f7bee47d217c0c365f80ce45de.